reader_test.go

```diff
@@ -21,34 +21,72 @@ func TestConsumerMaxWaitExceeded(t *testing.T) {
 	defer writer.Close()
 
 	// Create a reader to consume messages.
-	assert.NotPanics(t, func() {
-		reader := test.module.Kafka.reader(&ReaderConfig{
-			Brokers: []string{"localhost:9092"},
-			Topic:   test.topicName,
-			MaxWait: Duration{time.Second * 3},
-		})
-		assert.NotNil(t, reader)
-		defer reader.Close()
+	reader := test.module.Kafka.reader(&ReaderConfig{
+		Brokers: []string{"localhost:9092"},
+		Topic:   test.topicName,
+		MaxWait: Duration{250 * time.Millisecond},
```

Comment:
We do not need 3 seconds to get down to the `consume()` call even on noisy Github :)